pkg/snapshot/generator/postgres/schema/pgdumprestore/snapshot_pg_dump_restore_generator_test.go

```diff
@@ -455,6 +456,60 @@ func TestSnapshotGenerator_CreateSnapshot(t *testing.T) {
 
 			wantErr: nil,
 		},
+		{
+			name:              "ok - no roles dump",
```

Comment:
Should add a test for no passwords mode to confirm we're passing the right options to the pgdump all.